app/models/concerns/roleable.rb

```diff
@@ -20,8 +20,6 @@ def regular?
 
   def whisperer?
     @whisperer ||= begin
-      return false if !SiteSetting.enable_whispers?
-      return true if staff?
```

Comment:
So `staff` don't get a bypass anymore? Is that the only place where that assumption was made/hardcoded?

Reply:
Even the old bypass was bypassing only half of the restrictions. Whispers had to be enabled in order for group check to be bypassed for staff members. I did my best to migrate this in the database migration.

Reply:
Depends: do we still want the `staff` bypass?

Reply:
I think we could add it back just to make this change less surprising and remove the DB migration.